mochi/.gitignore

```diff
@@ -0,0 +1,12 @@
+/ebin
+/doc
+/_test
+/.eunit
+/docs
+.DS_Store
+/TEST-*.xml
+/deps
+/.rebar
+*.swp
+*.beam
+*.dump
```

Comment:
Hmm, phoenix (elixir-lang) beats them all in time per request and transfer
rate, but never gets above 21k req/s.

It's even more bloaty once it's done generating the server skeleton, stupid
templating, channels, db layers and prod deployment code...

Concurrency Level:      10
Time taken for tests:   0.490 seconds
Complete requests:      10000
Failed requests:        0
Write errors:           0
Total transferred:      2250000 bytes
HTML transferred:       50000 bytes
Requests per second:    20412.75 [#/sec] (mean)
Time per request:       0.490 [ms] (mean)
Time per request:       0.049 [ms] (mean, across all concurrent requests)
Transfer rate:          4485.22 [Kbytes/sec] received

On Sun, May 31, 2015 at 9:39 AM, Eskil Olsen <eskil@eskil.org> wrote:

> Just autogenerated code from creating a service. Think playdoh/potter.
>
> On Fri, May 29, 2015 at 2:56 PM, Aiden Scandella <notifications@github.com
> > wrote:
>
>> In mochi/.gitignore
>> <https://github.com/sectioneight/http-shootoff/pull/3#discussion_r31368304>
>> :
>>
>> > @@ -0,0 +1,12 @@
>> > +/ebin
>> > +/doc
>> > +/_test
>> > +/.eunit
>> > +/docs
>> > +.DS_Store
>> > +/TEST-*.xml
>> > +/deps
>> > +/.rebar
>> > +*.swp
>> > +*.beam
>> > +*.dump
>>
>> BLOAT
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/sectioneight/http-shootoff/pull/3/files#r31368304>.
>>
>
>
>
> --
> eskil
> ----
>



-- 
eskil
----